ontoma/EFO_Handler.py

```diff
@@ -0,0 +1,177 @@
+"""Class to handle EFO ontology related functions."""
```

Comment:
PEP8 says packages should have all-lowercase names (although underscores are permitted): https://peps.python.org/pep-0008/#package-and-module-names

Reply:
I didn't put much though into the names. I'm happy to change.